### PR TITLE
Set up dryrun for all the pr projects

### PR DESCRIPTION
**Pull request by krzyzacy**
Try to observe the log and make next step

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/912)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/jenkins-janitor.yaml [hunk 1]
```diff
@@ -25,4 +25,14 @@
         - activate-gce-service-account
         - shell: |
             python $WORKSPACE/jenkins/janitor.py --project=sen-lu-test --hours=1
+            python $WORKSPACE/jenkins/janitor.py --project=kubernetes-pr-cri-validation --hours=6 --dryrun=true
```
Comment (fejta) [inline jenkins/job-configs/kubernetes-jenkins/jenkins-janitor.yaml]: Let's have the default be safe? Aka add --dryrun=false for sen-lu-test and the others are true by default?

Comment (krzyzacy) [inline jenkins/job-configs/kubernetes-jenkins/jenkins-janitor.yaml]: Which job shall I refer to to use bootstrap? Seems I don't need a template one.. Is there a document for bootstrap?

**Comment by krzyzacy** [comment]
woops... I guess I rebased wrong and it wiped my PR

**Comment by fejta** [comment]
Ouch :cry:

**Comment by ixdy** [comment]
As long as you haven't `git gc`'d it should be recoverable.

**Pull request closed without merging**